src/constants/config.ts

```diff
@@ -266,3 +266,8 @@ export const TERM_OF_SERVICE = 'https://www.tidebit-defi.com/term_of_service/{ha
 export const PRIVATE_POLICY = 'https://www.tidebit-defi.com/private_policy/{hash}';
 export const DeWT_VALIDITY_PERIOD = 60 * 60; // seconds
 export const CFD_LIQUIDATION_TIME = 86400;
+export const SELL_PRICE_ERROR = 0;
```

Comment:
- remove and rename these constant params
  - SELL_PRICE_ERROR -> DEFAULT_SELL_PRICE
  - BUY_PRICE_ERROR -> DEFAULT_BUY_PRICE
  - USER_BALANCE_ERROR -> DEFAULT_BALANCE_AMOUNT